Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM python:3.12
+# Use a Python image with uv pre-installed
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm
+
+# Enable bytecode compilation
+ENV UV_COMPILE_BYTECODE=1
+
 WORKDIR /usr/local/app/DashApp
 
-COPY requirements.txt ../requirements.txt
+COPY pyproject.toml ../pyproject.toml
+COPY uv.lock ../uv.lock
```

Comment:
Do we need to copy the lock file over? or does this get generated by `uv sync` 4 lines later? I haven't got hands on experience with `uv` yet to know!

Reply:
Ah right I'm with you. I thought `sync` was installing the packages but now I see it's not.